### ISSUE TITLE
BulkRead: only read certain properties

### ISSUE THREAD

**Issue by SIT-jekleine** [borisdj/EFCore.BulkExtensions#590]
Is it possible to to only read cetain properties with the BulkRead function? I am trying to find a solution to load a single property based on the Id of the entity.

**Comment by borisdj** [comment]
You can use **_PropertiesToInclude_** or **_PropertiesToInclude_**, more info in ReadMe.

**Comment by SIT-jekleine** [comment]
> You can use **_PropertiesToInclude_** or **_PropertiesToExclude_**, more info in ReadMe.

That is indeed what I was trying to do without any success. 
The function call:
![image](https://user-images.githubusercontent.com/87364324/125439812-f8e74673-e09f-47d1-9e9c-850b7b1d62dc.png)

And the actual sql query being executed where AddressableId shouldn't be included:
![image](https://user-images.githubusercontent.com/87364324/125439918-da8fe3a0-b903-41cd-915b-a57f9207ed35.png)

**Comment by borisdj** [comment]
You are correct, only in memory excluded are  skipped but from Db it selects all.
Will consider if this could be improved.

**Comment by sommmen** [comment]
Would also love this, useful to reload ids only.

**Comment by sommmen** [comment]
> You are correct, only in memory excluded are skipped but from Db it selects all. Will consider if this could be improved.

Perhaps you could give me a quick description of what'd need to be done?  I might take a peek if its not too complicated and i have some spare time during the weekends.

**Comment by oresttkachukd** [comment]
Is there a chance it was working in version 3.1?
